### ISSUE TITLE
PP is not always updated correctly after using moves

### ISSUE THREAD

**Issue by Rangi42** [Rangi42/polishedcrystal#400]
Maniac379: "I just used Shadow Claw on my Marowak against rivals Haunter and it did not consume a PP use.  This seems to happen now and then."

**Comment by FredrIQ** [comment]
Cannot reproduce

**Comment by Rangi42** [comment]
This could be related to a report by Eternity:

"This screenshot is after battling a trainer with 3 pokemons. I used 2 Leech Lifes and 1 Shadow Claw. End of battle and this is the state"

![image](https://user-images.githubusercontent.com/35663410/81480493-324c4e80-91f8-11ea-8346-6da88a02de33.png)

"I noticed something similar when battling the E4. My Exeggutor didn't use Protect a single time, but before I went against Lance I checked and it was 6/10 pps"

"It happened again. I was in Noisy Forest. I was using Exeggutor a while and then switched to Bellossom. This happened (I used a single Petal Dance to finish the battle)"

![image](https://user-images.githubusercontent.com/35663410/81480514-47c17880-91f8-11ea-8c44-c33ece745b7d.png)
